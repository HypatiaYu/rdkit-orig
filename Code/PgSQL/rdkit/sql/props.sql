SELECT mol_amw('c1ccccc1'::mol) mol_amw;
SELECT mol_logp('c1ccccc1'::mol) mol_logp;
SELECT mol_hba('c1ccccc1'::mol) mol_hba;
SELECT mol_hbd('c1ccccc1'::mol) mol_hbd;
SELECT mol_hba('c1ccncc1'::mol) mol_hba;
SELECT mol_hbd('c1ccncc1'::mol) mol_hbd;
SELECT mol_hbd('c1ccncc1O'::mol) mol_hbd;
SELECT mol_hba('c1ccncc1O'::mol) mol_hba;
SELECT mol_logp('c1ccncc1O'::mol) mol_logp;
SELECT mol_numrotatablebonds('CCC'::mol) mol_numrotatablebonds;
SELECT mol_numrotatablebonds('CCCC'::mol) mol_numrotatablebonds;
SELECT mol_numrotatablebonds('c1ccccc1c1ccc(CCC)cc1'::mol) mol_numrotatablebonds;
SELECT mol_numheavyatoms('CCC'::mol) val;
SELECT mol_numatoms('CCC'::mol) val;
SELECT mol_numheteroatoms('CCC'::mol) val;
SELECT mol_numheteroatoms('CCO'::mol) val;
SELECT mol_tpsa('CCC'::mol) val;
SELECT mol_tpsa('CCO'::mol) val;
SELECT mol_numrings('CCC'::mol) val;
SELECT mol_numrings('C1CC1'::mol) val;
