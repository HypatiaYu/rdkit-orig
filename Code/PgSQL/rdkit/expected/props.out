SELECT mol_amw('c1ccccc1'::mol) mol_amw;
 mol_amw 
---------
  78.114
(1 row)

SELECT mol_logp('c1ccccc1'::mol) mol_logp;
 mol_logp 
----------
   1.6866
(1 row)

SELECT mol_hba('c1ccccc1'::mol) mol_hba;
 mol_hba 
---------
       0
(1 row)

SELECT mol_hbd('c1ccccc1'::mol) mol_hbd;
 mol_hbd 
---------
       0
(1 row)

SELECT mol_hba('c1ccncc1'::mol) mol_hba;
 mol_hba 
---------
       1
(1 row)

SELECT mol_hbd('c1ccncc1'::mol) mol_hbd;
 mol_hbd 
---------
       0
(1 row)

SELECT mol_hbd('c1ccncc1O'::mol) mol_hbd;
 mol_hbd 
---------
       1
(1 row)

SELECT mol_hba('c1ccncc1O'::mol) mol_hba;
 mol_hba 
---------
       2
(1 row)

SELECT mol_logp('c1ccncc1O'::mol) mol_logp;
 mol_logp 
----------
   0.7872
(1 row)

SELECT mol_numrotatablebonds('CCC'::mol) mol_numrotatablebonds;
 mol_numrotatablebonds 
-----------------------
                     0
(1 row)

SELECT mol_numrotatablebonds('CCCC'::mol) mol_numrotatablebonds;
 mol_numrotatablebonds 
-----------------------
                     1
(1 row)

SELECT mol_numrotatablebonds('c1ccccc1c1ccc(CCC)cc1'::mol) mol_numrotatablebonds;
 mol_numrotatablebonds 
-----------------------
                     3
(1 row)

SELECT mol_numheavyatoms('CCC'::mol) val;
 val 
-----
   3
(1 row)

SELECT mol_numatoms('CCC'::mol) val;
 val 
-----
  11
(1 row)

SELECT mol_numheteroatoms('CCC'::mol) val;
 val 
-----
   0
(1 row)

SELECT mol_numheteroatoms('CCO'::mol) val;
 val 
-----
   1
(1 row)

SELECT mol_tpsa('CCC'::mol) val;
 val 
-----
   0
(1 row)

SELECT mol_tpsa('CCO'::mol) val;
  val  
-------
 20.23
(1 row)

SELECT mol_numrings('CCC'::mol) val;
 val 
-----
   0
(1 row)

SELECT mol_numrings('C1CC1'::mol) val;
 val 
-----
   1
(1 row)

SELECT mol_murckoscaffold('c1ccccc1CCC'::mol) val;
   val    
----------
 c1ccccc1
(1 row)

